components/eamxx/src/physics/gw/impl/gw_gwd_compute_tendencies_from_stress_divergence_impl.hpp

```diff
@@ -20,27 +27,150 @@ void Functions<S,D>::gwd_compute_tendencies_from_stress_divergence(
   const Int& pgwv,
   const Int& ngwv,
   const bool& do_taper,
-  const Spack& dt,
-  const Spack& effgw,
+  const Real& dt,
+  const Real& effgw,
   const uview_1d<const Int>& tend_level,
-  const uview_1d<const Spack>& lat,
-  const uview_1d<const Spack>& dpm,
-  const uview_1d<const Spack>& rdpm,
-  const uview_1d<const Spack>& c,
-  const uview_1d<const Spack>& ubm,
-  const uview_1d<const Spack>& t,
-  const uview_1d<const Spack>& nm,
-  const uview_1d<const Spack>& xv,
-  const uview_1d<const Spack>& yv,
+  const uview_1d<const Real>& lat,
```

Comment:
It's fine to go with Real here if you've decided that at the top level you'll take all views from the AD as Real arrays rather than pack arrays.

Reply:
I haven't decided that. I was hoping someone would know for sure if we want to do that or not. Using `Real` definitely makes the code easier.

Reply:
My opinion is that it's fine for individual parameterizations to decide not to use packs.